modules/import_all.py

```diff
@@ -3,134 +3,155 @@
 from __future__ import absolute_import
 
 import argparse
-from copy import deepcopy
-from datetime import datetime
-from hashlib import sha256
 import json
 import logging
 import re
+from copy import deepcopy
+from functools import partial
+from hashlib import sha256
 
 import apache_beam as beam
+from apache_beam.io import WriteToBigQuery
 from apache_beam.options.pipeline_options import PipelineOptions
-from apache_beam.runners import DataflowRunner
-
-from modules import constants, utils
-
-
-# BigQuery can handle rows up to 100 MB.
-MAX_CONTENT_SIZE = 10 * 1024 * 1024
-# Number of times to partition the requests tables.
-NUM_PARTITIONS = 4
-
-
-def partition_step(function, har, client, crawl_date, index):
-    """Partitions functions across multiple concurrent steps."""
 
-    if not har:
-        logging.warning('Unable to partition step, null HAR.')
-        return
-
-    page = har.get('log', {}).get('pages', [{}])[0]
-    page_url = page.get('_URL')
-
-    if not page_url:
-        logging.warning('Skipping HAR: unable to get page URL (see preceding warning).')
-        return
+from modules import constants, utils, transformation
+from modules.transformation import (
+    ReadHarFiles,
+    add_common_pipeline_options,
+    HarJsonToSummary,
+)
 
-    hashed_url = hash_url(page_url)
-    if hashed_url % NUM_PARTITIONS != index:
-        return
+# BigQuery can handle rows up to 100 MB when using `WriteToBigQuery.Method.FILE_LOADS`
+MAX_CONTENT_SIZE_100_MB = 10 * 1024 * 1024
+# BigQuery can handle rows up to 100 MB when using `WriteToBigQuery.Method.STREAMING_INSERTS`
+MAX_CONTENT_SIZE_10_MB = 1024 * 1024
```

Comment:
Should these be enums?

I'm also confused why they're named 100 when the value is 10, and 10 when the value is 1.